### PR TITLE
docs: organize and edit FAQ

### PR DESCRIPTION
**Pull request by TheCatLady**
#### Description

Organizes FAQs into appropriate categories, as well as some minor edits.

#### To-Dos

N/A

#### Issues Fixed or Closed

N/A

### CODE REVIEW

File: docs/support/faq.md [hunk 1]
```diff
@@ -1,46 +1,54 @@
...
-**Troubleshooting Steps:**
+- New Plex Movie
+- Legacy Plex Movie
+- New Plex TV
+- Legacy Plex TV
+- TheTVDB
+- TMDb
+- HAMA
 
-First, check the Overseerr logs for media items that are missing. The logs will contain an error as to why that item could not be matched. One example might be `errorMessage":"SQLITE_CONSTRAINT: NOT NULL`. This means that the TMDb ID is missing from the Plex XML for that item.
```
Comment (samwiseg0) [inline docs/support/faq.md:43]: Why was this removed? We should leave this in so people can search based off the error.

File: docs/support/faq.md [hunk 2]
```diff
@@ -58,44 +66,58 @@ You can also perform the following to verify the media item has a GUID Overseerr
...
    4. Legacy Plex Movie agent `guid="com.plexapp.agents.imdb://tt0765446"`
 
-### TV series requests are failing after I updated Overseerr!
-
-**A:** Language profile support for Sonarr was added in [#860](https://github.com/sct/overseerr/pull/860), along with a new "Language Profile" required setting. If your TV series requests are failing, please make sure that you have a default language profile configured for each of your Sonarr servers in **Settings &rarr; Services**.
-
 ### Where can I find the logs?
 
-**A:** The logs are located at `<Overseeerr-install-directory>/logs/overseerr.log`
+Log files are located in `<Overseeerr config directory>/logs/`.
```
Comment (samwiseg0) [inline docs/support/faq.md:71]: Can we leave `overseerr.log` in there? Maybe lets just link to asking for support page so we only keep this in one place? https://docs.overseerr.dev/support/asking-for-support#how-can-i-share-my-logs

**Review by samwiseg0 — CHANGES_REQUESTED** [review]

File: docs/support/faq.md (hunk 1 above)
Comment (sct) [reply docs/support/faq.md:43]: This error shouldn't be possible anymore.

Comment (TheCatLady) [reply docs/support/faq.md:43]: > Why was this removed? We should leave this in so people can search based off the error.

sct had asked me to remove it. 😸

File: docs/support/faq.md [hunk 3]
```diff
@@ -1,46 +1,54 @@
...
+Overseerr currently supports the following agents:
 
-**Troubleshooting Steps:**
+- New Plex Movie
+- Legacy Plex Movie
+- New Plex TV
+- Legacy Plex TV
+- TheTVDB
+- TMDb
+- HAMA
```
[the remark this answers is not in the document]
Comment (TheCatLady) [reply docs/support/faq.md]: Added.  (https://github.com/ZeroQI/Hama.bundle is the right URL, right?)

File: docs/support/faq.md (hunk 2 above)
Comment (TheCatLady) [reply docs/support/faq.md]: Fixed!

**Review by samwiseg0 — APPROVED** [review]
